libshaderc_util/CMakeLists.txt

```diff
@@ -25,7 +25,7 @@ target_include_directories(shaderc_util
   PUBLIC include PRIVATE ${glslang_SOURCE_DIR})
 find_package(Threads)
 target_link_libraries(shaderc_util PRIVATE
```

Comment:
Good question.  I think it should be  PUBLIC here.  In fact I think Glslang's CMakeLists.txt should be fixed to export them as public as well.  No?